tools/build_memory_info.py

```diff
@@ -52,7 +52,7 @@
 with open(sys.argv[1], "r") as f:
     for line in f:
         line = line.strip()
-        if line.startswith(("FLASH_FIRMWARE", "FLASH", "RAM")):
+        if line.startswith(("FLASH_FIRMWARE", "RAM")):
```

Comment:
While we're here... is there a particular reason why we show only the remaining free bytes, rather than simply the amount of flash used? I use this metric the most often to gauge the results of optimization or to see whether it will fit in a low-flash board or a certain sector on a board, but I have always had to reverse this operation to get that info - I have never needed to know the remaining amount of flash as a raw number. Maybe that's more interesting to others, though?

Reply:
I often need to know just how tight it is, so it sounds like both numbers are useful

Reply:
"330104 bytes used, 718472 bytes free out of 1048576 in flash firmware space"
"330104 bytes used out of 1048576 in flash firmware space, 718472 bytes free"

Reply:
+print("{} bytes used, {} bytes free in flash firmware space out of {} bytes ({}kB).".format(used_flash, free_flash, firmware_region, firmware_region / 1024))